flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/canal/CanalJsonDeserializationSchema.java

```diff
@@ -223,6 +224,8 @@ public void deserialize(byte[] message, Collector<RowData> out) throws IOExcepti
 					insert.setRowKind(RowKind.DELETE);
 					out.collect(insert);
 				}
+			} else if (OP_CREATE.equals(type)){
+				return;
```

Comment:
Please add a comment what is an CREATE message, and why we just ignore it.

Reply:
done